workos/typing/enums.py

```diff
@@ -0,0 +1,48 @@
+from enum import Enum
+from typing import Any, TypeVar, Union
+from typing_extensions import Annotated
+from pydantic import (
+    Field,
+    ValidationError,
+    ValidationInfo,
+    ValidatorFunctionWrapHandler,
+    WrapValidator,
+)
+from workos.typing.untyped_literal import UntypedLiteral
+
+
+EnumType = TypeVar("EnumType", bound=Enum)
+
+# Identical to the literal or strings path, except meant for a literal of enum values.
```

Comment:
Should I just delete this from the PR? Not being used at the moment.

Reply:
Yep, can delete.